partner_address_format/models.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+
+from openerp import models, fields, api
+from openerp.addons.base.ir.ir_qweb import QWebException
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    address_format_id = fields.Many2one('ir.ui.view',
+                                        help='A normal view with the format to'
+                                             'be used to format the address '
+                                             'for this company.')
+
+
+class ResCountry(models.Model):
+    _inherit = 'res.country'
+
+    address_format_id = fields.Many2one('ir.ui.view',
+                                        help='A normal view with the format to'
```

Comment:
`s/A normal view with the format to format/A normal view with the template to format/g`